Sources/UIAlertController.swift

```diff
@@ -63,6 +63,12 @@ public class UIAlertController: UIViewController {
         )
     }
 
+    public override func viewWillLayoutSubviews() {
+        let size = view.bounds.size
+        self.alertControllerView?.center = CGPoint(x: size.width / 2,y: size.height / 2)
+        super.viewWillLayoutSubviews()
```

Comment:
is this order necessary? Otherwise I would prefer calling `super.viewWillLayoutSubviews` first